conf/nomad/trino.hcl

```diff
@@ -218,8 +218,17 @@ ${hive_config_properties}
 EOH
         destination = "secrets/trino/hive.properties"
       }
+%{ if use_memory_connector }
```

Comment:
Could this possibly be split out into a separate file? Will Nomad collate files?
Not yet: https://github.com/hashicorp/nomad/issues/7954

Reply:
> Looks good, but some changes required.
> 
> There must be links to documentation added, and possibly also a simple example on how/why to use?

I was thinking its own section would be too much, and therefore only added it to the input doc. There is no mention of the other connectors (Trino and PostgreSQL) either. 

If we add doc for the memory connector we should add doc for all connectors under a "Connectors" section. WDYT?

Reply:
You want this to make everything more organised and clearer, correct? (also, can I resolve this?)